changelogs/current.yaml

```diff
@@ -5,6 +5,12 @@ behavior_changes:
 
 minor_behavior_changes:
 # *Changes that may cause incompatibilities for some users, but should not for most*
+- area: ratelimit
```

Comment:
This is a new feature rather than a minor behavior change? If the config remains the same then the behavior doesn't change at all.

Reply:
yes, thanks, PTAL again.